docs/library/picodrive.md

```diff
@@ -117,14 +124,14 @@ If you have e.g. `foo.bin`, you should create a text file and save it as `foo.cu
    INDEX 01 00:00:00
 ```
 
-After that, you can load the `foo.cue` file in RetroArch with the PicoDrive core.
+After that, you can load the `foo.cue` file in RetroArch with the Genesis Plus GX core.
```

Comment:
Why was this changed to genplusgx?

Reply:
Ah, I was using the Genesis Plus GX  doc as a template and forgot to replace that. I pushed a commit to fix it.